diff --git a/node_modules/graphql-tools/dist/stitching/schemaRecreation.js b/node_modules/graphql-tools/dist/stitching/schemaRecreation.js
index 3709771..980c737 100644
--- a/node_modules/graphql-tools/dist/stitching/schemaRecreation.js
+++ b/node_modules/graphql-tools/dist/stitching/schemaRecreation.js
@@ -62,7 +62,9 @@ function recreateType(type, resolveType, keepResolvers) {
         var newValues_1 = {};
         values.forEach(function (value) {
             newValues_1[value.name] = {
-                value: value.value,
+                // Simply pass-through the name of the enum entry such that the enum type in the original schema will
+                // receive that and will be able to do its work as if it were never proxied.
+                value: value.name,
                 deprecationReason: value.deprecationReason,
                 description: value.description,
                 astNode: value.astNode,
